File: NetFlexAPI/NetFlexAPI/Models/Serial.cs
namespace NetFlexAPI.Models;

public class Serial
{
    public Guid Id { get; set; }
    public string Poster { get; set; }
    public string Title { get; set; }
    public int NumEpisodes { get; set; }
    public int AgeRating { get; set; }
    public float UserRating { get; set; }
    public string Description { get; set; }
}

File: NetFlexAPI/GiraffeAPI/Models/Serial.fs
module GiraffeAPI.Models.Serial

open System
open System.ComponentModel.DataAnnotations
open Microsoft.EntityFrameworkCore.Metadata.Internal

[<CLIMutable>]  
type Serial =
    {
        Id : Guid
        Poster : string
        Title : string
        NumEpisodes : int
        AgeRating : int
        UserRating : float
        Description : string
    }
    
    